src/vortex.py

```diff
@@ -0,0 +1,17 @@
+class Vortex:
+    def __init__(self):
+        self.SECRETS = {}   # Populate with sources
+        self.endpoints = {} # Populate with sinks
```

Comment:
A bit late to the party but does this mean you'll store all the secret keys you find on this dictionary? Make extra sure it doesn't leak itself haha

Reply:
The idea is to have the user access secrets programmtically instead of `os.env` calls, kinda like https://github.com/spf13/viper
It won't leak anything unless the user does, and in the job of our application is to detect that anyway :smile: